VAMobile/documentation/docs/App Features/WhatsNew/WhatsNew.md

```diff
@@ -0,0 +1,18 @@
+# Whats New
+
+## Feature Summary
+
+Whats new displays to the veteran what is new in the version that they upgraded to. This alert only shows if the the version of the app the user is on is the most recent in the store and if there is an important enough change to annouce with the provided notes to display to the user. The user can dismiss this alert from reappearing for the current version.
+
+## Use Cases
+
+* Use Case 1: The user is on the most recent version available in the store and there is notes for the alert to display resulting in the alert displaying(see screenshot)
+* Use Case 2: The user is on the most recent version with notes to display, but the user has dismissed the alert causing it to no longer be displayed
+* Use Case 3: The user is on a older version of the app and instead sees [Encouraged Update](../EncouragedUpdate/EncouragedUpdate.md)
```

Comment:
```suggestion
* Use Case 3: The user is on an older version of the app and instead sees [Encouraged Update](../EncouragedUpdate/EncouragedUpdate.md)
```

Reply:
added

Reply:
@Sparowhawk Looks like this suggestion wasn't applied